src/components/calcite-input/calcite-input.tsx

```diff
@@ -176,7 +176,7 @@ export class CalciteInput {
     | "week" = "text";
 
   /** input value */
-  @Prop({ mutable: true, reflect: true }) value?: string = "";
+  @Prop({ mutable: true }) value: string;
```

Comment:
nitpick: Is this optional or required?

`@Prop({ mutable: true }) value?: string;`

or

`@Prop({ mutable: true }) value!: string;`

Reply:
Optional.